src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -264,7 +264,7 @@
     text: |
         az acr import -n MyRegistry --source sourcerepository:sourcetag -t targetrepository:targettag \\
             -r /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/sourceResourceGroup/providers/Microsoft.ContainerRegistry/registries/sourceRegistry
-  - name: Import an image and return right after starting an import.
+  - name: Import an image without waiting for completion.
```

Comment:
@adewaleo I like this idea. Since imageImport does not provide a GET API that users could leverage to check their import status, it is a good user experience to warn them of this if they opt out of waiting. We can recommend them in the warning to run `az acr repository show-tags` to confirm the import succeeded (however this does not cover the case where import overwrites tags). @nathana1 what do you think?

`--no-wait=True` will still catch failures in the front end of the request after it lands in our RP and before we send the request to the backend for processing. These errors (request validation, etc) are all thrown before we create the async response. Failure in the backend to actually perform the import will not be surfaced.

Reply:
I think adding a warning to the description/documentation would be a good idea.  I don't think we'd want to add a warning to every time the command is run with this flag.

Reply:
yeah I was thinking more along the line of documentation warning. (I do think logging a warning on `--no-wait` imports could also be helpful)